os/tools/mkbinheader.py

```diff
@@ -50,24 +52,31 @@
 binary_name = sys.argv[4]
 binary_ver = sys.argv[5]
 dynamic_ram_size = sys.argv[6]
+comp_enabled = sys.argv[7]
+comp_blk_size = sys.argv[8]
```

Comment:
You are not using comp_blk_size.

Reply:
It is being used.
Please check changes  in code under "if bin_comp > COMP_NONE :"

Reply:
However, it is only used if we are compressing the binary (comp_enabled > 0).
Else, if we are not compressing binary, it is not being used.